nav2_msgs/action/NavigateThroughPoses.action

```diff
@@ -22,3 +22,4 @@ builtin_interfaces/Duration estimated_time_remaining
 int16 number_of_recoveries
 float32 distance_remaining
 int16 number_of_poses_remaining
+WaypointStatus[] waypoint_statuses
```

Comment:
I think this should be in the `result` too so that its both on feedback for live updates but as well in the final result of the task